fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkWriter.java

```diff
@@ -236,9 +236,10 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
 
             // Pushes to the Outcome Site
             if (collectImportOperationOutcomes) {
-                Provider wrapper = ProviderFactory.getSourceWrapper(ctx.getOutcome(), "ibm-cos");
-                wrapper.registerTransient(chunkData);
-                wrapper.pushOperationOutcomes();
+                Provider provider = ProviderFactory.getSourceWrapper(ctx.getOutcome(),
+                    ConfigurationFactory.getInstance().getStorageProviderType(ctx.getOutcome()));
```

Comment:
Not in this case, since it's using the outcome storageType.

Reply:
ok, I was confused since on line 164 "StorageType type = adapter.getStorageProviderStorageType(ctx.getOutcome());" it looked to be the exact same thing.

Reply:
Ah, ok.  I misread it, getStorageProviderStorageType and getStorageProviderType looked the same when I first looked at it.